ci/spelling-config.json

```diff
@@ -21,6 +21,12 @@
         "spiffe",
         "SPIFFE",
         "keycloak",
-        "Keycloak"
+        "Keycloak",
+        "triage",
+        "Triage",
```

Comment:
In my experience, the spell-check is case-insensitive.  (I was surprised, for example, that acronyms needed to be added here, but also see how that's useful.)

If someone more familiar with cSpell or we can find reference to documentation and can confirm, we should remove spiffe/SPIFFE redundancy as well.

cc @JohnHillegass @php-coder

Reply:
it caught both `keycloak` and `Keycloak` when i ran it.

Reply:
Yeah @ultrasaurus it should be case-insensitive, so we only need to enter the words once. @TheFoxAtWork can you try updating to just have one entry and see if it works as expected?

More info here https://github.com/streetsidesoftware/cspell/tree/main/packages/cspell#things-to-note